autorest/codegen/models/client.py

```diff
@@ -57,6 +53,23 @@ def _imports_shared(self, async_mode: bool) -> FileImport:
             file_import.add_submodule_import(
                 "typing", "Optional", ImportType.STDLIB, TypingSection.CONDITIONAL
             )
+        if self.code_model.options["client_side_validation"]:
```

Comment:
you should be able to add a helper function for on `FileImport`, something like `add_msrest_import` so this code can be shared